storybook/src/docs/typography.docs.mdx

```diff
@@ -114,11 +115,11 @@ The text sizes at those points are as follows, rounded to 1 decimal place:
 
 #### Compact
 
-| Device Class | Design Width |    0 |    1 |    2 |    3 |    4 |    5 |    6 |
-| :----------- | ------------ | ---: | ---: | ---: | ---: | ---: | ---: | ---: |
-| Phone        | 320          | 25.2 | 22.4 | 19.9 | 17.7 | 15.8 |   14 | 12.4 |
-| Tablet       | 832          | 29.0 | 25.3 | 22.1 | 19.3 | 16.9 | 14.8 |   13 |
-| Desktop      | 1600         | 34.6 | 29.6 | 25.4 | 21.8 | 18.7 |   16 | 13.7 |
+| Device Class | Design Width |    0 |    1 |    2 |    3 |    4 |   5 |    6 |
+| :----------- | ------------ | ---: | ---: | ---: | ---: | ---: | --: | ---: |
+| Phone        | 320          | 22.1 | 20.7 | 19.4 | 18.2 | 17.1 |  16 |   15 |
```

Comment:
Do we need to specify that the unit of measurement is pixel?

Reply:
We mention the word ‘pixel’ eleven times on this page, maybe not in this section, but I think it’s clear in general.